tictactoe/lib/tictactoe/game.ex

```diff
@@ -23,7 +46,7 @@ defmodule Tictactoe.Game do
   end
 
   defp mark_board(game, position, mark) do
-    Map.put(game, :board, List.replace_at(game.board, position - 1, mark))
+    %{ game | board: List.replace_at(game.board, position - 1, mark) }
```

Comment:
It seems that the `game` has a lot of responsibility here:
- board logic
- creating the players
- running the game